modules/features/settings/src/main/res/layout/fragment_settings_appearance.xml

```diff
@@ -233,6 +234,7 @@
                 android:layout_marginTop="4dp"
                 android:layout_marginEnd="24dp"
                 android:contentDescription="@string/settings_use_episode_artwork"
+                android:saveEnabled="false"
```

Comment:
Added because of race conditions between restoring state and setting the setting from view model. It is ok to not save state for this view because it is fully controlled from the view model.